### PR TITLE
allows a user to fetch host code after the meeting has been created

### LINKED ISSUE #183: Add feature to retrieve host code if you were the one who created a /z
Does it make sense to allow users to re-request the host code as it is delivered via an ephemeral msg and sometimes can be lost?  Haven't thought this one through but I have heard of plenty of ppl "losing it" over the past few months

### PR DESCRIPTION
**Pull request by cskartikey**
Should close #183

### CODE REVIEW

File: api/endpoints/slack/host-code.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+  const meeting = await Prisma.find('meeting', { where: { zoomID: text } });
+
+  if (!meeting) {
+    return sendEphemeralMessage(response_url, 'Unable to retrieve the host code. Please check the code and remove any Markdown formatting.');
+  }
+
+  if (meeting.endedAt) {
+    return sendEphemeralMessage(response_url, 'Unable to retrieve the host code for a concluded meeting.');
```
Comment (JosiasAurel) [inline api/endpoints/slack/host-code.js:43]: I think 'Cannot retrieve the host code for a concluded meeting.' will be better

File: api/endpoints/slack/host-code.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+  if (meeting.creatorSlackID !== user_id) {
+    return sendEphemeralMessage(response_url, '_You can only retrieve the host code for meetings you created._');
+  }
+  
+  if (meeting.creatorSlackID === user_id) {
+    return sendEphemeralMessage(response_url, `_Your meeting code is: *${meeting.hostKey}*_`);
+  }
```
Comment (JosiasAurel) [inline api/endpoints/slack/host-code.js:52]: does this need to be wrapped in an if block? if all other conditions pass we should be able to just return the ephemeral message right?

**Review by JosiasAurel — APPROVED** [review]
overall looks good to me

**Comment by cskartikey** [comment]
Good point! Will amend accordingly.

**Review by JosiasAurel — APPROVED** [review]
lgtm

**Comment by cskartikey** [comment]
- [ ] Modify the manifest to reflect the new command

**Pull request merged**